xbmc/platform/darwin/osx/network/NetworkOsx.cpp

```diff
@@ -180,27 +181,45 @@ std::vector<std::string> CNetworkOsx::GetNameServers()
 {
   std::vector<std::string> result;
 
-  FILE* pipe = popen("scutil --dns | grep \"nameserver\" | tail -n2", "r");
-  usleep(100000);
+  FILE* pipe = popen("scutil --dns | grep \"nameserver\" | uniq | tail -n2", "r");
```

Comment:
probably a micro-optimization, but we could also use `grep -F nameserver`

also, why only last 2 are picked? In my case I'll be seeing only IPv6 servers:

```
  nameserver[0] : 1.1.1.1
  nameserver[1] : 1.0.0.1
  nameserver[2] : 2606:4700:4700::1111
  nameserver[3] : 2606:4700:4700::1001
```

Reply:
Yeah we have two infos `NETWORK_DNS1_ADDRESS` and `NETWORK_DNS2_ADDRESS` which are populated to the first and second configured DNS servers.
Added the `-F` option although I doubt it has any noticeable difference.